scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -52,6 +49,21 @@ if [ "${ARCH}" == "s390x" ]; then
     make test-cmd-watch
     # E2e tests
     make test-e2e-beta
+elif  [ "${ARCH}" == "ppc64le" ]; then
```

Comment:
Why are we listing these individually for ppc

Reply:
There were/are some flaky/inconsistent issues observed for ppc64le ( for e.g. https://github.com/openshift/odo/issues/3454 ), hence purposely kept a separate block for ppc64le, so that if a few of these particular test suites fail consistently can comment them and proceed while issues are being resolved. Going ahead will merge this block into one as things become stable.

Reply:
Ok

Reply:
Closing this as per below comment from @mohammedzee1000